### PR TITLE
UHM-5772: Move HIV and EID program enrollment from sections

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

**Comment by mogoodrich** [comment]
https://pihemr.atlassian.net/browse/UHM-5772

File: configuration/pih/htmlforms/hiv/section-hiv-assessment.xml [hunk 1]
```diff
@@ -108,6 +108,14 @@
         </script>
     </ifMode>
 
+    <!-- Program enrollment (if the patient hasn't already been enrolled); use first ancestor location tagged as Program Location:
+        note that we already do this on the history section, but add here case the history section is skipped
+        the special case for infants (less than 2 years old) is only handled on the history section, as it prompts the user for a question of whether to enroll in EID or HIV
+    -->
+    <includeIf velocityTest="$patient.getAge($encounter.getEncounterDatetime()) &gt; 1">
+        <enrollInProgram programId="HIV" locationTag="Program Location"/>
+    </includeIf>
```
Comment (mseaton) [inline configuration/pih/htmlforms/hiv/section-hiv-assessment.xml:117]: Hmm, could run into problems in the future when we forget we need to keep in sync these two sections.

File: configuration/pih/htmlforms/hiv/section-hiv-plan.xml [hunk 2]
```diff
@@ -89,6 +89,14 @@
...
 
+    <!-- Program enrollment (if the patient hasn't already been enrolled); use first ancestor location tagged as Program Location:
+    note that we already do this on the history section, but add here case the history section is skipped
+    the special case for infants (less than 2 years old) is only handled on the history section, as it prompts the user for a question of whether to enroll in EID or HIV
+-->
+    <includeIf velocityTest="$patient.getAge($encounter.getEncounterDatetime()) &gt; 1">
+        <enrollInProgram programId="HIV" locationTag="Program Location"/>
+    </includeIf>
+
     <ifMode mode="VIEW" include="false">
```
Comment (mseaton) [inline configuration/pih/htmlforms/hiv/section-hiv-plan.xml:100]: Make that 3 sections we need to keep in sync.

**Comment by mseaton** [comment]
Some high-level comments.  I thought we were only putting this on the plan?

**Comment by mogoodrich** [comment]
@mseaton  I was going to add it to each section that was Haiti-specific, which is just plan and assessment.  We could add it just to plan, or not add it at all...

**Review by mseaton — APPROVED** [review]
Approve, but note my comments

**Pull request merged**